src/components/ChartComponent.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import HrfChart from './HrfChart';
+
+const ChartContents = styled.div`
```

Comment:
Let's move all Styled Components out of the component and import them. Cleaner and makes it so that style adjustments don't require going into business logic.